clusterloader2/pkg/prometheus/manifests/default/kube-scheduler-service.yaml

```diff
@@ -1,15 +0,0 @@
-apiVersion: v1
```

Comment:
I think that this approach has the benefit of working both in GCE and GKE while the one from this PR won't work in GKE when it comes to system pods like kube-scheduler or kube-controller-manager, right?